pkg/opentelemetry/config.go

```diff
@@ -5,82 +5,186 @@ import (
 	"fmt"
 	"time"
 
+	k6Const "go.k6.io/k6/lib/consts"
 	"go.k6.io/k6/output"
+	"gopkg.in/guregu/null.v3"
 )
 
 const (
-	grpcReceiverType = "grpc"
+	// grpcExporterType GRPC exporter type
+	grpcExporterType = "grpc"
+	// httpExporterType HTTP exporter type
+	httpExporterType = "http"
 )
 
 // Config is the config for the template collector
 type Config struct {
+	// ServiceName is the name of the service to use for the metrics
+	// export, if not set it will use "k6"
+	ServiceName string
+	// ServiceVersion is the version of the service to use for the metrics
+	// export, if not set it will use k6's library version
+	ServiceVersion string
 	// MetricPrefix is the prefix to use for the metrics
 	MetricPrefix string
-	// ReceiverType is the type of the receiver to use
-	ReceiverType string
-	// GRPCReceiverEndpoint is the endpoint of the gRPC receiver
-	GRPCReceiverEndpoint string
-	// PushInterval is the interval at which to push metrics to the receiver
-	PushInterval time.Duration
 	// FlushInterval is the interval at which to flush metrics from the k6
 	FlushInterval time.Duration
+
+	// ExporterType sets the type of OpenTelemetry Exporter to use
+	// Currently only "grpc" is supported
```

Comment:
Don't we support `http`?  🤔 If not, why are all the HTTP references used for?

Reply:
Good catch :+1: It's a leftover, will remove this line :+1:

Reply:
I think the line in the validation method confirms it: 

```
if c.ExporterType != grpcExporterType && c.ExporterType != httpExporterType {
```

So I guess we can just remove that old comment 😛